<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Authors_books</class>
 <widget class="QWidget" name="Authors_books">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>551</width>
    <height>284</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>551</width>
    <height>284</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="maximumSize">
          <size>
           <width>141</width>
           <height>16</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Segoe Script</family>
          </font>
         </property>
         <property name="text">
          <string>Книги автора</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="ChengeAuthor">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="font">
          <font>
           <family>Segoe Script</family>
           <pointsize>12</pointsize>
          </font>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="maximumSize">
              <size>
               <width>163</width>
               <height>25</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Segoe Script</family>
              </font>
             </property>
             <property name="text">
              <string>Выбранная книга:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="ChengeBook">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="minimumSize">
              <size>
               <width>231</width>
               <height>25</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Segoe Script</family>
              </font>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="show_book">
             <property name="maximumSize">
              <size>
               <width>153</width>
               <height>33</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Segoe Script</family>
              </font>
             </property>
             <property name="text">
              <string>Показать книгу</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QColumnView" name="columnView"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="back">
         <property name="maximumSize">
          <size>
           <width>75</width>
           <height>23</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Segoe Script</family>
          </font>
         </property>
         <property name="text">
          <string>Назад</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
